src/pages/api/sendEmail.ts

```diff
@@ -0,0 +1,48 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import nodemailer from 'nodemailer';
+
+type Data = {
+  message?: string;
+  error?: string;
+  details?: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  const { to, subject, text } = req.body;
+
+  // Validate the input
+  if (!to || !subject || !text) {
+    return res.status(400).json({ error: 'Missing required fields' });
+  }
+
+  // Create a transporter object
+  try {
+    const transporter = nodemailer.createTransport({
+      service: process.env.EMAIL_SERVICE,
+      auth: {
+        user: process.env.EMAIL_USER,
+        pass: process.env.EMAIL_PASS, // app password
+      },
+    });
+
+    // Configure the mailOptions object
+    const mailOptions = {
+      from: process.env.EMAIL_USER,
+      to: to,
+      subject: subject,
+      text: text,
```

Comment:
It may be useful to also add `html: text` here.
According [this page](https://nodemailer.com/message/) of the Nodemailer docs, it also supports HTML. I think eventually, we would want to enable the admin to style things as well.
@ChinemeremChigbo could you please confirm that this is within the scope? if not, this comment could be resolved directly.

Reply:
I think the scope of this ticket was just to set up nodemailer and send an email, and we'll deal with email format later because as of right now there's no specification of what the email should look like